### PR TITLE
docs: improve README examples of lib/node_modules/@stdlib/constants/u…

### LINKED ISSUE #1566: [RFC]: improve README examples of `constants/unicode` namespace
### Description

This RFC proposes to improve the [README examples](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/constants/unicode#examples) of the `constants/unicode` namespace package. Currently, namespace package READMEs only include minimal examples which are not particularly informative and do not showcase namespace functionality.

Historically, namespace examples were not given much effort due to the rapidly changing nature of the project; however, now that development and organization has settled, it would be good to revisit these packages and add proper examples to better communicate to users how to use namespace contents.

For those wanting to contribute to this issue, you can use the following steps:

- Browse through the [contents](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/constants/unicode) of the `constants/unicode` namespace and identify interesting functions and packages to showcase.
- Update the example code block in the "Examples" section of the README for the `constants/unicode` namespace and open a PR.
- When updating the README, examples should follow existing project conventions (e.g., ES5).
- Examples may `require` functionality from other packages to help make examples more interesting.

### Related Issues

None.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by BeastMasterGrinder**
Resolves #1566 .

## Description

> improve README examples of constants/unicode namespace

## Related Issues

> No

This pull request:

-   resolves #1566 
-   fixes #1566 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [ ] Read, understood, and followed the [contributing guidelines][contributing].
-   [ ] Searched for existing issues and pull requests.
-   [ ] The issue name begins with RFC:

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/constants/unicode/README.md [hunk 1]
```diff
@@ -22,21 +22,33 @@ limitations under the License.
 
 > Unicode constants.
 
-<section class="usage">
```
Comment (kgryte) [inline lib/node_modules/@stdlib/constants/unicode/README.md:25]: Why is this removed?

File: lib/node_modules/@stdlib/constants/unicode/README.md [hunk 2]
```diff
@@ -22,21 +22,33 @@ limitations under the License.
...
 var constants = require( '@stdlib/constants/unicode' );
+
+// Maximum Unicode code point:
+console.log( 'Maximum Unicode code point:', constants.MAX );
+
+// Maximum Basic Multilingual Plane (BMP) code point:
+console.log( 'Maximum BMP code point:', constants.MAX_BMP );
 ```
 
-#### constants
```
Comment (kgryte) [inline lib/node_modules/@stdlib/constants/unicode/README.md:33]: Why was this change made? Where else in the project do we make the API heading level 2?

File: lib/node_modules/@stdlib/constants/unicode/README.md [hunk 3]
```diff
@@ -22,21 +22,33 @@ limitations under the License.
...
 ## Usage
 
 ```javascript
 var constants = require( '@stdlib/constants/unicode' );
+
+// Maximum Unicode code point:
+console.log( 'Maximum Unicode code point:', constants.MAX );
+
+// Maximum Basic Multilingual Plane (BMP) code point:
+console.log( 'Maximum BMP code point:', constants.MAX_BMP );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/constants/unicode/README.md:34]: The first code block is always the import. Please see other packages in the project.

**Review by kgryte — CHANGES_REQUESTED** [review]
This PR makes several changes we don't want and does not conform to existing project conventions/practices. As such, we won't be able to move this forward until changes are made.

**Comment by Planeshifter** [comment]
Closing due to inactivity.

**Pull request closed without merging**